autoscale.polo

```diff
@@ -0,0 +1,32 @@
+---
+- schema_version: 1.0.0
+  name: Autoscale
+  aliases:
+    - autoscaling
+    - otter
+  summary: Rackspace Autoscale
+  description:
+  source: https://github.com/rackerlabs/otter
+  tracker: https://github.com/rackerlabs/otter/issues
+  website: https://one.rackspace.com/display/Autoscale/Auto+Scale
+  environments:
+    - tier: production
+      default: true
+      infrastructure: public
+      datacenters:
+        - dfw
+        - ord
+        - iad
+        - lon
+        - hkg
+        - syd
+      aliases:
+        - autoscale.api.rackspacecloud.com
+        - dfw.autoscale.api.rackspacecloud.com
+        - ord.autoscale.api.rackspacecloud.com
+        - iad.autoscale.api.rackspacecloud.com
+        - lon.autoscale.api.rackspacecloud.com
+        - hkg.autoscale.api.rackspacecloud.com
+        - syd.autoscale.api.rackspacecloud.com
+      dependencies:
+        - https://identity.api.rackspacecloud.com/v2.0
```

Comment:
We depend on many other services: compute, CLB, Rackconnect, Cloud feeds, Heat and Metrics. Should I add those too?